pkg/apis/core/types.go

```diff
@@ -4350,6 +4350,27 @@ type PodStatus struct {
 	// +featureGate=DynamicResourceAllocation
 	// +optional
 	ResourceClaimStatuses []PodResourceClaimStatus
+
+	// AllocatedResources is the total requests allocated for this pod by the node.
+	// Kubelet sets this to the accepted requests when a pod (or resize) is admitted.
```

Comment:
I think the API server should set this before the pod is bound (or maybe just on create), so let's drop this implementation detail.